File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/entity/Credit.java
package org.nirvanasms.api.core.entity;

public class Credit {
	private int amount;
	
	public Credit(){}
	
	public Credit(int amount){
		this.amount = amount;
	}

	public int getAmount() {
		return amount;
	}

	public void setAmount(int amount) {
		this.amount = amount;
	}
	
}

File: NirvanaSMS.Api.CSharp/Core/Entity/Credit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NirvanaSMS.Api.Core.Entity
{
    public class Credit
    {
        public int Amount { get; set; }

        public Credit()
        {

        }

        public Credit(int amount)
        {
            Amount = amount;
        }
    }
}
